invenio.cfg

```
@@ -445,3 +445,8 @@ RDM_RECORDS_IDENTIFIERS_SCHEMES = {**RDM_RECORDS_IDENTIFIERS_SCHEMES,
                                       "inspire": {"label": _("Inspire"),
                                                   "validator": schemes.is_inspire,
                                                   "datacite": "INSPIRE"}}}
+
+RDM_RECORDS_PERSONORG_SCHEMES = {**RDM_RECORDS_PERSONORG_SCHEMES,
+                                 **{"inspire": {"label": _("Inspire"),
```

I think the name is capital letters:
```suggestion
                                 **{"inspire": {"label": _("INSPIRE"),
```